foodsaving/groups/migrations/0002_group_members.py

```diff
@@ -16,9 +17,32 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='GroupMembership',
+            fields=[
+                ('id', models.AutoField(primary_key=True, serialize=False)),
+            ],
+            options={
+                'db_table': 'groups_group_members',
+            },
+        ),
         migrations.AddField(
             model_name='group',
             name='members',
-            field=models.ManyToManyField(to=settings.AUTH_USER_MODEL),
+            field=models.ManyToManyField(related_name='groups', through='groups.GroupMembership', to=settings.AUTH_USER_MODEL),
```

Comment:
This change pretends that there was never a version _without_ the `through` model. Makes sense to me!

Reply:
Shhhhh don't tell anyone. It's always been this way... ;)